contracts/utils/contracts/src/DeploymentConstants.sol

```diff
@@ -26,22 +26,30 @@ contract DeploymentConstants {
     // address constant private WETH_ADDRESS = 0xd0A1E359811322d97991E03f863a0C30C2cF029C;
     /// @dev Mainnet address of the KyberNeworkProxy contract.
     address constant private KYBER_NETWORK_PROXY_ADDRESS = 0x818E6FECD516Ecc3849DAf6845e3EC868087B755;
+    // /// @dev Kovan address of the KyberNeworkProxy contract.
+    // address constant private KYBER_NETWORK_PROXY_ADDRESS = 0x692f391bCc85cefCe8C237C01e1f636BbD70EA4D;
     /// @dev Mainnet address of the `UniswapExchangeFactory` contract.
     address constant private UNISWAP_EXCHANGE_FACTORY_ADDRESS = 0xc0a47dFe034B400B47bDaD5FecDa2621de6c4d95;
+    // /// @dev Kovan address of the `UniswapExchangeFactory` contract.
+    // address constant private UNISWAP_EXCHANGE_FACTORY_ADDRESS = 0xD3E51Ef092B2845f10401a0159B2B96e8B6c3D30;
     /// @dev Mainnet address of the Eth2Dai `MatchingMarket` contract.
     address constant private ETH2DAI_ADDRESS = 0x39755357759cE0d7f32dC8dC45414CCa409AE24e;
+    // /// @dev Kovan address of the Eth2Dai `MatchingMarket` contract.
+    // address constant private ETH2DAI_ADDRESS = 0x4A6bC4e803c62081ffEbCc8d227B5a87a58f1F8F   ;
     /// @dev Mainnet address of the `ERC20BridgeProxy` contract
     address constant private ERC20_BRIDGE_PROXY_ADDRESS = 0x8ED95d1746bf1E4dAb58d8ED4724f1Ef95B20Db0;
     // /// @dev Kovan address of the `ERC20BridgeProxy` contract
     // address constant private ERC20_BRIDGE_PROXY_ADDRESS = 0xFb2DD2A1366dE37f7241C83d47DA58fd503E2C64;
     ///@dev Mainnet address of the `Dai` (multi-collateral) contract
     address constant private DAI_ADDRESS = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    // ///@dev Kovan address of the `Dai` (multi-collateral) contract
+    // address constant private DAI_ADDRESS = 0x4F96Fe3b7A6Cf9725f59d353F723c1bDb64CA6Aa;
     /// @dev Mainnet address of the `Chai` contract
```

Comment:
What is the convention here when you deploy to kovan you uncomment?

Reply:
yeah, pretty high tech